src/core/toType.js

```diff
@@ -0,0 +1,25 @@
+define( function() {
+	"use strict";
+
+	var class2type = {};
+
+	// Populate the class2type map
+	jQuery.each( "Boolean Number String Function Array" +
```

Comment:
You need to add `../core` to the dependencies as `jQuery`.

Reply:
Hmm, I was also debating changing this to just hardcode the values. I think min/gziped it's even smaller. Then we won't have the dependency on `../core`. (I had this change locally but decided to revert it for this pr, which probably broke some things...).